README.md

```diff
@@ -1,12 +1,12 @@
-# Ably PHP Laravel Wrapper
+# Simple Ably-PHP Laravel Wrapper
```

Comment:
I'm not sure if I like this, I think the title should stay the same and instead we should use markdown to add a visible warning just below the h1, like:

> **Warning**
> If you're using Laravel and want to support **realtime broadcasting and events**, it is recommended to use [ably/laravel-broadcaster](https://packagist.org/packages/ably/laravel-broadcaster/) instead.

Reply:
Updated!